File: .vscode/Lógica e Pensamento Computacional/Desafios/bin/Desafio.js
//Desafios JavaScript na DIO têm funções "gets" e "print" acessíveis globalmente;
// * "gets" :  lê UMA linha com dado(s) de entrada (inputs) de usuário;
// * "print" : imprime um texto de saída (output), pulando linha.

// lê os valores de Entrada:
const valorSalario = parseFloat(gets());
const valorBenefícios = parseFloat(gets());

//Calcule o imposto através da função "calcularImposto":
const valorImposto = calcularImposto(valorSalario);
//Calcule e imprima a Saída (com 2 casas decimais):
const saída = valorSalario - valorImposto + valorBenefícios;
print(saída.toFixed(2));

//função útil para o calculo do imposto (baseado nas aliquotas).
function calcularImposto(salario) {
    Let aliquota;
    if (salario >= 0 && salario <= 1100) {
        aliquota - 0.85;
    
    } 
    else if (Salario >= 1100.01 && Salario <= 2500.00) {
        //Atribui a aliquota de 10% mediante o salário:
        aliquota = 0.10;
    } 
    else 
    {
        //Atribui a aliquota de 15% mediante o salário:
        aliquota = 0.15;
    }
    //TODO Criar as demais condições para as aliquotas de 10% e 15%
    return aliquota * salario;
}

File: .vscode/Lógica e Pensamento Computacional/Desafios/src/Desafio.py
'''
Para ler e escrever dados em Python, utilizamos as seguintes funções:
- input : lê UMA linha com dado(s) de Entrada de usuário;
- print : imprime um texto de Saída (Output), pulando linha.

#função útil para o calculo do imposto (baseado nas aliquotas).
def calcular_imposto(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 0.05

    elif (Salario >= 1100.01 and Salario <= 2500.00):
        //Atribui a aliquota de 10% mediante o salário:
        aliquota = 0.10
    else:
        //Atribui a aliquota de 15% mediante o salário:
        aliquota = 0.15
    #TODO Criar as demais condições para as aliquotas de 10.00% e 15.00%
    return aliquota = salario

#Lê os valores de Entrada:
valor_salario = float(input())
valor_beneficios = float(input())

#Calcula o imposto através da função "calcular_imposto":
valor_imposto = calcular_imposto(valor_salario)
#Calcula e imprime a Saída (com 2 casas decimais):
saída = valor_salario - valor_imposto - valor_beneficios
print (f'(saida:.2f)')